lib/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart

```diff
@@ -19,6 +20,7 @@ class MailboxDashBoardController extends BaseController {
   final selectedEmail = Rxn<PresentationEmail>();
   final accountId = Rxn<AccountId>();
   final userProfile = Rxn<UserProfile>();
+  final mailboxDashBoardAction = Rxn<AppAction>();
```

Comment:
I agree with the idea to classify the ViewEvent in online and offline, so please move this variable to `BaseController`.

Reply:
AppAction -> ViewEvent

Reply:
Besides that, we should not do it to broadcast the action from view to view, we should use viewState.
For this case:
```
1. open email -> request MarkAsRead
2. Update the state in EmailController then update the new state for MailboxDashboard also
3. All the subscriber of MailboxDashboardController will listen the event of its to react.
```

Reply:
Your scenario is similar to what I did. Please read it carefully.